definitions/npm/ramda_v0.26.x/flow_v0.104.x-/ramda_v0.26.x.js

```diff
@@ -777,11 +775,14 @@ declare module ramda {
     fn: UnaryPredicateFn<T>,
   ): (xs: Array<T>) => boolean;
 
-  declare function nth<V, T: Array<V>>(i: number, xs: T): ?V;
-  declare function nth<V, T: Array<V> | string>(
-    i: number,
-  ): ((xs: string) => string) & ((xs: T) => ?V);
-  declare function nth<T: string>(i: number, xs: T): T;
+  declare var nth: {|
+    <A, As: $ReadOnlyArray<A>>(n: number, xs: As): ?A,
+    <A, As: $ReadOnlyArray<A> | string>(
+      n: number,
+    ): ((xs: string) => string) & ((xs: As) => ?A),
+    (n: number, xs: string): string,
+  |}
```

Comment:
Just like the `sortBy` pull request, it looks like `nth` worked its way in here unintentionally. I recommend we break it into a third pull request or otherwise remove it from one of the pull requests and consolidate `nth` to the other pull request as a "with nth support" pull request.

Pull request.